packages/fx-core/src/common/m365/launchHelper.ts

```diff
@@ -38,7 +38,9 @@ export class LaunchHelper {
         let installAppPackage = true;
         if (
           capabilities.length > 0 &&
-          capabilities.filter((capability) => !copilotCapabilities.includes(capability)).length == 0
+          (capabilities.filter((capability) => !copilotCapabilities.includes(capability)).length ==
+            0 ||
+            capabilities.includes("apiMeAAD"))
```

Comment:
just want to double confirm：
- if the app has both tab and apiMeAAD capability, should we skip installAppPackage or direct user to installAppPackage URL?

Reply:
Thanks, I updated the condition. It will only skip installAppPackage if it doesn't include a tab and it's an API ME project with AAD auth.